rpcs3/Emu/Audio/audio_resampler.h

```diff
@@ -6,6 +6,7 @@
 #ifndef _MSC_VER
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wsuggest-override"
+#pragma GCC diagnostic ignored "-Wold-style-cast"
```

Comment:
why

Reply:
There is an old style cast in SoundTouch which leads to a compilation error.